lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
Alright, in that case I was thinking maybe we should move the adb forward to before phase.setup also, what do you think? If adb forward fails does that affect logging the start mark?